.gitignore

```diff
@@ -8,7 +8,6 @@
 /examples/sample-app/logs/openshift.log
 *.swp
 .vimrc
-.kubeconfig
```

Comment:
I suppose that's true… just didn't see the relevance to the pull, and didn't consider it too crufty